Misc/NEWS.d/3.12.0a1.rst

```diff
@@ -4559,8 +4559,7 @@ The minimum Sphinx version required to build the documentation is now 3.2.
 .. section: Documentation
 
 Augmented documentation of asyncio.create_task(). Clarified the need to keep
-strong references to tasks and added a code snippet detailing how to to
-this.
+strong references to tasks and added a code snippet detailing how to this.
```

Comment:
May as well fix the grammar here, while we're here

```suggestion
strong references to tasks and added a code snippet detailing how to do this.
```

Reply:
Yes of course. Sorry for my inattention